src/store/modules/mobile.js

```diff
@@ -1,8 +1,8 @@
 import Vue from 'vue'
 import uuid from 'uuid/v4'
-import { Crypto } from '@aeternity/aepp-sdk'
+import { Crypto } from '@aeternity/aepp-sdk/src'
```

Comment:
the examples from `aeternity/aepp-sdk-js/docs/usage.md` use `'@aeternity/aepp-sdk/es'` (**es** instead of **src**). Is that intentional to use the `src` directly?

Reply:
Yes, the Base app currently depends on an old version of `aepp-sdk` where source code placed in `src` directory.

Reply:
ok!